python/requests/best-practice/use-raise-for-status.yaml

```
@@ -14,6 +14,14 @@ rules:
       $RESP.raise_for_status(...)
   - pattern-not-inside: |
       requests.$METHOD(...).raise_for_status(...)
+  - pattern-not-inside: |
+      $RESP = requests.$METHOD(...)
+      if $RESP.status_code == ...:
+        ...
+  - pattern-not-inside: |
+      $RESP = requests.$METHOD(...)
+      if $RESP.status_code != ...:
+        ...
```

```suggestion
        ...
  - pattern-not-inside: |
      $RESP = requests.$METHOD(...)
      if $RESP.ok:
        ...
  - pattern-not-inside: |
      $RESP = requests.$METHOD(...)
      if not $RESP.ok:
        ...
```

This is Drew's suggestion I think.